packages/plugin-ext/src/main/browser/text-editor-main.ts

```diff
@@ -162,6 +167,11 @@ export class TextEditorMain {
                 lineNumbers: lineNumbers
             });
         }
+
+        this.editor.getControl().updateOptions({
```

Comment:
it should not be necessary, preferences are already hooked up with Monaco editors: https://github.com/theia-ide/theia/blob/5c67bdfe6d625046d10bdf875b67b28ed12a6c9b/packages/monaco/src/browser/monaco-editor-provider.ts#L106-L109

Reply:
thank you @akosyakov, I had trouble understanding earlier how it was tied together